protostar/cli/protostar_arg_type_test.py

```diff
@@ -51,3 +63,24 @@ def test_fail_on_wrong_block_explorer():
     parser = map_protostar_type_name_to_parser("block_explorer")
     with pytest.raises(ValueError):
         parser("abc")
+
+
+@pytest.fixture(name="fake_contract_path")
+def fake_contract_path_fixture(tmp_path: Path) -> Path:
+    contract_path = tmp_path / "main.cairo"
+    contract_path.touch()
+    return contract_path
+
+
+def test_contract_source_identifier_parser(fake_contract_path: Path):
+    parser = create_map_protostar_type_name_to_parser(
+        ContractSourceIdentifierFactory(
+            FakeConfigurationFile(
+                contract_name_to_source_paths={"main": [fake_contract_path]},
+            )
+        )
+    )("contract_source_identifier")
+
+    result = parser("main")
+
+    return result == ContractSourceIdentifier(name="main", paths=[fake_contract_path])
```

Comment:
Isn't this test basically running the code tested to assess correctness of the result?

Reply:
[fix assert](https://github.com/software-mansion/protostar/pull/1185/commits/d58fccfdd0e600ce90f11298cd2d6c51c04dc4f1)